docs/application/dotnet/guides/nui/nui-components/Pagination.md

```diff
@@ -4,135 +4,222 @@ Pagination is a common component supporting multiple pages. It shows the number
 
 ![Pagination](./media/Pagination.png)
 
+
+In the following sections on this guide one can find:
+- [the main Pagination properties](#paginationProperties)
+- how to create a Pagination
+   - [using Properties](#createWithProperty)
+   - [using Style](#createWithStyle)
+   - [using a custom style](#createWithCustomStyle)
+- [how to connect Pagination with a KeyEvent](#connectWithKeyEvent)
+
+
+<a name="paginationProperties"></a>
+## Pagination properties
+
+The table below lists the properties specific for the Pagination class.
+
+**Table: Pagination properties**
+
+| Property                  | Type               | Description                                   |
+|---------------------------|--------------------|-----------------------------------------------|
+| `IndicatorSize`           | `Size`             | The absolute size of the indicator.           |
+| `IndicatorSpacing`        | `int`              | The space between the indicators (in pixels). |              
+| `IndicatorCount`          | `int`              | The count of the indicators/pages.            |              
```

Comment:
I only left 'The count of the indicators' because it is the user's responsibility to link pagination with the pages and I didn't want it to look like it was automatic

Reply:
Are you ok with 'The count of the indicators' ?
I don't like this 'or', since it won't happen automatically, the user has to identify the indicators with the pages.
If you wish I add the removed part.

Reply:
@mgrudzinska ok, then what is an appropriate word there? What do you wish to convey?

Reply:
The count of the indicators'
I think it's clear enough. Do you agree?